File: Ambulance.BLL/Models/MedicalRecordModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AmbulanceSystem.BLL.Models;
public class MedicalRecordModel
{
    public int MedicalRecordId { get; set; }
    public int MedicalCardId { get; set; }
    public int BrigadeMemberId { get; set; }
    public DateTime? CreationDate { get; set; }
    public string? Diagnoses { get; set; }
    public string? Symptoms { get; set; }
    public string? Treatment { get; set; }
    public string? ImageUrl { get; set; }
    public string? BrigadeMemberName { get; set; }
    public string? MedicalCardPatientName { get; set; }
}

File: ambulance-site/src/api/models/MedicalRecordModel.js
export class MedicalRecordModel {
    constructor({
        medicalRecordId = 0,
        medicalCardId = 0,
        brigadeMemberId = 0,
        dataTime = null,
        diagnoses = null,
        symptoms = null,
        treatment = null,
        imageUrl = null,
        brigadeMemberName = null,
        medicalCardPatientName = null
    } = {}) {
        this.medicalRecordId = medicalRecordId;
        this.medicalCardId = medicalCardId;
        this.brigadeMemberId = brigadeMemberId;
        this.dataTime = dataTime ? new Date(dataTime) : null;
        this.diagnoses = diagnoses;
        this.symptoms = symptoms;
        this.treatment = treatment;
        this.imageUrl = imageUrl;
        this.brigadeMemberName = brigadeMemberName;
        this.medicalCardPatientName = medicalCardPatientName;
    }
}